perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -139,6 +139,13 @@ public static boolean authorized(PerunSession sess, String policyDefinition, Lis
 			refreshAuthz(sess);
 		}
 
+		if (System.currentTimeMillis() - sess.getPerunPrincipal().getRolesUpdatedAt() >= (long) BeansUtils.getCoreConfig().getRoleUpdateInterval() * 60000) {
```

Comment:
I know that this is really easy math, but IMHO it can be more readable with something like this: TimeUnit.MINUTES.toMillis(_myMinutes_) - it also returns long, so you don't need to change the type then;
```suggestion
		if (System.currentTimeMillis() - sess.getPerunPrincipal().getRolesUpdatedAt() >= TimeUnit.MINUTES.toMillis(BeansUtils.getCoreConfig().getRoleUpdateInterval())) {
```